api/src/org/labkey/api/dataiterator/LoggingDataIterator.java

```diff
@@ -34,7 +34,7 @@
  */
 public class LoggingDataIterator extends AbstractDataIterator implements ScrollableDataIterator, MapDataIterator
 {
-    static Logger _staticLog = LogManager.getLogger(LoggingDataIterator.class);
+    static Logger _staticLog = LogHelper.getLogger(LoggingDataIterator.class, "Shows transformations and mappings during many types of data imports and ETLs");
```

Comment:
```suggestion
    static Logger _staticLog = LogHelper.getLogger(LoggingDataIterator.class, "Transformations and mappings during many types of data imports and ETLs");
```
(Just for consistency.)